src/azure/cli/commands/_introspection.py

```diff
@@ -6,11 +6,24 @@
 import inspect
 import re
 
+def extract_full_summary_from_signature(operation):
+    """ Extract the summary from the doccomments of the command. """
+    lines = inspect.getdoc(operation)
+    regex = re.compile(r'\s*(:param)\s+(.+)\s*:(.*)')
```

Comment:
the regex can be pulled out of the method and compiled, then using the precompiled version will be faster

---
In reply to: [75970919](https://github.com/Azure/azure-cli/pull/736#discussion_r75970919) [](ancestors = 75970919)